_source/_posts/2021-11-08-k8s-api-server-oidc.md

```diff
@@ -0,0 +1,561 @@
+---
+layout: blog_post
+title: "How to Secure Your Kubernetes Cluster with OpenID Connect and RBAC"
+author: deepu-sasidharan
+by: advocate
+communities: [kubernetes, devops, security]
+description: "Learn how to set up Okta as an OpenID Connect provider using Terraform for access to Kubernetes API server."
+tags: [kubernetes, devops, oidc, okta, terraform]
+tweets:
+  - "Use #OIDC to access #Kubernetes API server securely. Set it up using Terraform."
+  - "Access #Kubernetes API server using your favorite #OIDC provider!"
+type: conversion
+image: blog/k8s-api-server-oidc/kube-login-oidc.jpg
```

Comment:
You can add a `github: /path/to/repo` line in the front matter here and it'll change the GitHub icon at the top to link to your repo instead of your username. I find it useful so I don't have to scroll to the bottom for the repo.

Reply:
You mean to put the terraform code repo here